distributed_index_tests/tests/test_dcp_reshard.py

```diff
@@ -0,0 +1,61 @@
+import time
+
+import concurrent.futures
+
+from lib.admin import Admin
+
+from cluster_setup import cluster
+
+
+def test_dcp_reshard(cluster):
+
+    cluster.reset("sync_gateway_default.json")
+
+    start = time.time()
+
+    admin = Admin(cluster.sync_gateways[2])
+    admin.register_user(target=cluster.sync_gateways[3], db="db", name="traun", password="password", channels=["ABC", "NBC", "CBS"])
```

Comment:
We should generate channel names programmatically. This should be array variable